source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,71 @@
+=======================================
+Accessing data of offline Users/Players 
+=======================================
+
+It may be necessary for plugins to access player data even when the player is offline.
+Since ``Sponge.getServer().getPlayer()`` does only return Players who are online one needs to find another solution.
+
+Some plugins store the relevant data themselves and associate the user by using ``Sponge.getServer().getGameProfileManager()``.
+But writing different code for offline and online users is not necessary:
+
+The Service Manager natively provides a service called ``UserStorageService`` which is capable of returning ``User`` instances for Players who are currently offline
+And since the ``Player`` interface extends ``User`` a ``User`` can often be used the same way as the ``Player`` instance.
```

Comment:
This doesn't entirely make sense... Because the `Player` interface extends `User`, you can use `User` as a `Player`???